.eslintrc.js

```diff
@@ -533,5 +533,7 @@ module.exports = {
     AsyncLocalStorage: 'readonly',
     async_hooks: 'readonly',
     globalThis: 'readonly',
+
+    RN$Bridgeless: 'readonly',
```

Comment:
Can you move this up to specify a specific file like `nativeFabricUIManager` does above? 

https://github.com/facebook/react/blob/main/.eslintrc.js#L419-L424